processor_test.go

```diff
@@ -271,6 +286,22 @@ func newTestService(c *Context) (Service, error) {
 	if err := ts.RegisterHandler(ts.ProcessMsg); err != nil {
 		panic(err.Error())
 	}
+
+	if err := ts.RegisterRESTHandler(procRestMsgGET1, testServiceName, "GET", 3, 3); err != nil {
```

Comment:
Those 3,3 out there on the end kind of bother me. I wonder if it would be more beautiful to have the default RegisterRESTHandler by varargs and default to 3,3 (hardcoded for now; to be changed in v4). Callers that care about versioning can opt-in by adding the min,max to the end). Dunno, this is not really important, now that I think about it.

Reply:
another way could be that the user can pass in a config object, 
```go
type ConfigREST struct {
    MaxVersion int
    MinVersion int
    // others if we need it, like xml support
}
```
if it's nil then we use some default